src/main/java/edu/fiuba/algo3/vistas/FactoryPreguntasVistas.java

```diff
@@ -0,0 +1,29 @@
+package edu.fiuba.algo3.vistas;
+
+import edu.fiuba.algo3.modelo.Kahoot;
+import javafx.scene.layout.Pane;
+
+public class FactoryPreguntasVistas {
+    public Pane crear(String tipoPregunta, Kahoot kahoot) {
+        Pane vista = null;
+
+        switch (tipoPregunta) {
+            case "VERDADERO_O_FALSO":
+                vista = new VerdaderoFalsoVista(kahoot);
+                break;
+            case "MULTIPLE_CHOICE":
+                /* vista =  new MultipleChoiceVista(kahoot);*/
+                break;
+            case "ORDERED_CHOICE":
+                /* vista = new OrderedChoiceVista(kahoot);*/
+                break;
+            case "GROUP_CHOICE":
+                /* vista = new GroupChoiceVista(kahoot);*/
+                break;
+            case "VERDADERO_O_FALSO_CON_PENALIDAD":
+                /* vista = new VerdaderoOFalsoConPenalidad(kahoot);*/
+                break;
```

Comment:
Ya que estamos, pueden reemplazar el switch por un diccionario (`HashMap`) donde la clave sea el string o enum y el valor la clase y usando "reflection" pueden instanciar la clase usando expresiones Lambda. 
- [Algo más de información](https://dzone.com/articles/factory-pattern-using-lambda-expression-in-java-8)
- [Repositorio de Sebastián mostrado el cuatri pasado](https://github.com/seblaz/Design-Patterns/blob/master/src/fiuba/algo3/factory/sol1/refactor1/Tablero.java)